TwoPaneLayout/library/src/main/java/com/microsoft/device/dualscreen/twopanelayout/twopanelayoutnav/TwoPaneNavScopeTest.kt

```diff
@@ -24,8 +24,13 @@ class TwoPaneNavScopeTest(
 
     override fun NavHostController.navigateTo(
         route: String,
-        screen: Screen,
-        navOptions: NavOptionsBuilder.() -> Unit
+        launchScreen: Screen,
+        builder: NavOptionsBuilder.() -> Unit
+    ) {
+    }
+
+    override fun NavHostController.navigateUpTo(
+        route: String?
     ) {
     }
 
```

Comment:
Can we move this class to the test package?

Reply:
This is part of our public API just so developers can write UI tests for any composables that need the TwoPaneScope/TwoPaneNavScope receiver! So it's not just for the library tests, but for anyone who is using TwoPaneLayout

https://learn.microsoft.com/dual-screen/android/jetpack/compose/two-pane-layout#test-twopanelayout-composables